### PR TITLE
LPD-41475 Technical Task | test-fix for Poshi tests

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPD-41475

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase [hunk 1]
```diff
@@ -238,7 +238,7 @@ definition {
 		}
 
 		task ("Then all fields of APISort object definition are present") {
-			var fieldNameList = "Author,Create Date,External Reference Code,ID,Modified Date,Status,OData Sort,UniversityAPISorts";
```
Comment (ccorreagg) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase:241]: I think this fix is already included in this PR: https://github.com/brianchandotcom/liferay-portal/pull/156050

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[APIRest]APIGenerator/ExposeQueryParametersInAPIExplorerForGeneratedAPI.testcase [hunk 2]
```diff
@@ -40,7 +40,7 @@ definition {
 	tearDown {
 		ApplicationAPI.deleteAllAPIApplication();
 
-		JSONObject.deleteAllCustomObjects();
```
Comment (ccorreagg) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[APIRest]APIGenerator/ExposeQueryParametersInAPIExplorerForGeneratedAPI.testcase:43]: Why do we need to include the pageSize? Do we have more than 20 objects? :exploding_head:

**Review by ccorreagg — COMMENTED** [review]
Hey! Just a couple of comments/questions

File: modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase (hunk 1 above)
Comment (magjed4289) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase:241]: I honestly don't understand how it got here, I created the branch from master and I don't see it as a change in this commit locally 🤯

Comment (magjed4289) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase:241]: yes, it is!

Comment (magjed4289) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase]: @ccorreagg updated

**Comment by magjed4289** [comment]
ci:test:headless-functional

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:help

**Comment by magjed4289** [comment]
ci:forward

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**